src/modules/edc-demo/components/property-grid/property-grid-field.ts

```diff
@@ -1,7 +1,8 @@
 export interface PropertyGridField {
   icon: string;
   label: string;
-  text: string;
+  text?: string;
```

Comment:
this should not be optional. An entry in the property grid should always have a value text. Right now the "guessValue" method adds the string '-' if there was no value present, and basically does this for all incoming values.